src/lib/comms/sol-oic-client.c

```diff
@@ -1126,31 +1160,31 @@ _poll_resource(void *data)
     return true;
 }
 
-static bool
+static int
 _observe_with_polling(struct sol_oic_client *client, struct sol_oic_resource *res,
     void (*callback)(void *data, sol_coap_responsecode_t response_code, struct sol_oic_client *cli, const struct sol_network_link_addr *addr,
     const struct sol_oic_map_reader *repr_vec),
     void *data)
 {
     struct resource_request_ctx *ctx = sol_util_memdup(&(struct resource_request_ctx) {
-        .client = client,
-        .cb = callback,
-        .data = data,
-        .res = res
-    }, sizeof(*ctx));
+            .base.client = client,
+            .cb = callback,
+            .base.data = data,
+            .res = res
+        }, sizeof(*ctx));
```

Comment:
I forgot to leave out these uncrustify v60 vs v61 issues: I'll fix them when merging.